lib/ansible/modules/cloud/amazon/iot_thing.py

```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
```

Comment:
The copyright header has changed:

http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright